### ISSUE TITLE
VQC documentation

### ISSUE THREAD

**Issue by sri1905** [Qiskit/qiskit-machine-learning#634]
### What should we add?

The methods are empty in the [VQC](https://qiskit.org/ecosystem/machinelearning/stubs/qiskit_machine_learning.algorithms.VQC.html#vqc) documentation. Also, the attributes are limited. Do the methods and attributes from the [stable version 0.32](https://qiskit.org/documentation/stable/0.32/stubs/qiskit.aqua.algorithms.VQC.html) still apply?

**Comment by woodsp-ibm** [comment]
The correct VQC reference is this https://qiskit.org/ecosystem/machine-learning/stubs/qiskit_machine_learning.algorithms.VQC.html (the first above generates a 404 error and the latter is a reference to an much older version in Qiskit Aqua which has been deprecated and unsupported for a couple of years now as the machine learning function was moved to this repo).

The lack of methods - empty list - is due to the fact that the class itself does not have any public methods. What the instance provides is from the parent classes. The documentation has been generated such that inherited methods are not replicated into every class that inherits them .

**Comment by adekusar-drl** [comment]
Yes, and methods from https://qiskit.org/ecosystem/machine-learning/stubs/qiskit_machine_learning.algorithms.SerializableModelMixin.html as well. Sorry, for inconvenience.

**Comment by adekusar-drl** [comment]
Shall I close the issue?

**Comment by adekusar-drl** [comment]
@sri1905 The documentation now includes inherited members as well. There's no need to go though full hierarchy. The updated documentation is available on the web site, e.g. VQC: https://qiskit.org/ecosystem/machine-learning/stubs/qiskit_machine_learning.algorithms.VQC.html
